pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/Namespaces.java

```diff
@@ -810,6 +811,42 @@ public void splitNamespaceBundle(@PathParam("property") String property, @PathPa
         }
     }
 
+    @PUT
```

Comment:
have updated with `POST` but,  [REST standard](http://restcookbook.com/HTTP%20Methods/put-vs-post) suggests `PUT` for resource-update and `POST` for create.